
---
uint8[] mask_red
